src/psyclone/nemo.py

```diff
@@ -327,3 +329,39 @@ def kernel(self):
                     f"{len(kernels)}")
             return kernels[0]
         return None
+
+
+class NemoACCEnterDataDirective(ACCEnterDataDirective):
```

Comment:
Where exactly should they be placed? Since `src/psyclone/domain/nemo` includes a `transformations` directory, should I create a `directives` directory too?
The fact they are the same was a very conscious design decision: I like the code of things that behave the same to look the same. How should I go about implementing `NemoDataMovementMixin`? You mean by keeping the `lower_to_language_level` specialisations for both classes and calling a static method in `NemoDataMovementMixin` to remove the loop variables?

Reply:
Done, `src/psyclone/domain/nemo/nodes` is a better place in my opinion though. Should I move them in this PR?

Reply:
No, let's leave them be.